landing_page/templates/landing_page/base.html

```diff
@@ -15,7 +15,9 @@
     />
     <link rel="stylesheet" href="{% static 'landing_page/css/index.css' %}" />
     <link rel="stylesheet" href="{% static 'landing_page/css/about.css' %}" />
+    <link rel="stylesheet" href="{% static 'landing_page/css/movie.css' %}" />
```

Comment:
this reference to the stylesheet should be in 'movie.html', not 'base.html'

Reply:
my bad, I started this precedent. habit to put it only in head element. 

But yeah, Elizabeth just place it above the html but below the django 'extends' line movie.html